js/filters.js

```diff
@@ -0,0 +1,61 @@
+const housingType = document.querySelector('#housing-type');
+const housingRoom = document.querySelector('#housing-rooms');
+const housingGuests = document.querySelector('#housing-guests');
+const housingPrice = document.querySelector('#housing-price');
+const features = document.querySelectorAll('[name=features]');
+
+const price = {
+  any:(value) => !value,
+  middle: (value) => value >= 10000 && value <= 50000,
+  low: (value) => value <= 10000,
+  high: (value) => value >= 50000,
+};
+
+function setOffersFilter(offers) {
+  const checkFn = price[housingPrice.value];
```

Comment:
следует указать что это для цены
```suggestion
  const checkPriceFn = price[housingPrice.value];
```